src/transaction.js

```diff
@@ -51,8 +42,9 @@ function mkNullDataReplayScript (
   )
 }
 
-/* More info: https://github.com/ZencashOfficial/zen/blob/master/src/script/standard.cpp#L377
+/*
  * Given an address, generates a pubkeyhash replay type script needed for the transaction
+ * More info: https://github.com/ZencashOfficial/zen/blob/bb93453d39f86f7889e87c50f06400427a66f816/src/script/standard.cpp#L403
```

Comment:
With time changes, link like `https://github.com/ZencashOfficial/zen/blob/master/src/script/standard.cpp#L377` will become outdated, 
Coping permalink helps us use solving the issue.
![image](https://user-images.githubusercontent.com/10692276/35191721-4f7f8330-fed6-11e7-816a-bf470bf0cffe.png)

Let me know if I linked the wrong line of code :)